spec/models/teacher_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+RSpec.describe Teacher, type: :model do
+  describe 'validations' do
+    subject { build(:teacher) }
+    it { should validate_presence_of(:first_name) }
```

Comment:
`should` works, but it's kinda outdated 😅 The current trend(well, for past several years) has been to use the `expectations` syntax like this:

```suggestion
    it { is_expected.to validate_presence_of(:first_name) }
```